sphinx_rtd_theme/theme.conf

```diff
@@ -10,8 +10,9 @@ collapse_navigation = True
 sticky_navigation = True
 navigation_depth = 4
 includehidden = True
+logo_background_color = 
```

Comment:
I think nav_header_background_color, sidebar_header_background_color or something like it would be better. This is more than just the background for the logo (also contains version and search) and there might not even be a logo

Reply:
Okay, I've changed it to nav_header_background_color